src/reducers/versions.spec.tsx

```diff
@@ -2301,26 +2301,85 @@ describe(__filename, () => {
         const position = pos as RelativePathPosition;
         expect(
           getRelativeDiffAnchor({
-            currentAnchor: '',
             diff,
             position,
           }),
         ).toEqual('D2');
       },
     );
 
-    it('throws an error if the currentAnchor cannot be found', () => {
+    it('returns the next anchor in the diff given a non-diff anchor', () => {
       const diff = createFakeDiffWithChanges([
-        [{ lineNumber: 1, type: 'insert' }],
+        [
+          { lineNumber: 1, type: 'delete' },
+          { lineNumber: 2, type: 'insert' },
+          { lineNumber: 3, type: 'normal' },
+          { lineNumber: 4, type: 'insert' },
+          { lineNumber: 5, type: 'normal' },
+          { lineNumber: 6, type: 'insert' },
+        ],
       ]);
-      const currentAnchor = 'D99';
-      expect(() => {
+      expect(
         getRelativeDiffAnchor({
-          currentAnchor,
+          currentAnchor: 'N3',
           diff,
           position: RelativePathPosition.next,
-        });
-      }).toThrow(`Could not locate anchor: ${currentAnchor} in the diff.`);
+        }),
+      ).toEqual('I4');
+    });
+
+    it('returns null if there is no next anchor in the diff given a non-diff anchor', () => {
+      const diff = createFakeDiffWithChanges([
+        [
+          { lineNumber: 1, type: 'delete' },
+          { lineNumber: 2, type: 'insert' },
+          { lineNumber: 3, type: 'normal' },
+        ],
+      ]);
+      expect(
+        getRelativeDiffAnchor({
+          currentAnchor: 'I2',
```

Comment:
A comment explaining that the delete took precedence over this insert might help to show how this is a non-diff anchor.